modules/mobile/spec/request/individual_lighthouse_claim_request_spec.rb

```diff
@@ -20,8 +20,9 @@
     before do
       token = 'abcdefghijklmnop'
       allow_any_instance_of(BenefitsClaims::Configuration).to receive(:access_token).and_return(token)
-      Flipper.enable(:mobile_lighthouse_claims)
-      iam_sign_in
+      user = build(:iam_user)
+      iam_sign_in(user)
+      Flipper.enable(:mobile_lighthouse_claims, user)
```

Comment:
Was this causing issues? I would expect it to work if the flag is on for everyone.

Reply:
yeah, the @current_user variable I called in the serializer was always nil but it didn't cause an error in specs since I just enabled it for every user anyways so it didn't need the @current_user variable.